client/src/components/UI/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./Carousel.scss";
+
+export default function Carousel(props) {
+  const { banners } = props;
+  var slideIndex = 0;
+  function showSlides() {
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    var dots = document.getElementsByClassName("dot");
+    if (slides.length > 0 && dots.length > 0) {
+      for (i = 0; i < slides.length; i++) {
+        slides[i].style.display = "none";
```

Comment:
use forEach instead directly on slides

Reply:
I can't use that because slides and dots are of type object.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Have a check for hasOwnProperty